test/controllers/purchases_controller_test.rb

```diff
@@ -1,45 +1,49 @@
-require "test_helper"
+require 'test_helper'
 
 class PurchasesControllerTest < ActionDispatch::IntegrationTest
   setup do
     @purchase = purchases(:one)
   end
 
-  test "should get index" do
+  test 'should get index' do
     get purchases_url
     assert_response :success
   end
 
-  test "should get new" do
+  test 'should get new' do
     get new_purchase_url
     assert_response :success
   end
 
-  test "should create purchase" do
-    assert_difference("Purchase.count") do
-      post purchases_url, params: { purchase: { amount: @purchase.amount, category_id: @purchase.category_id, name: @purchase.name, user_id: @purchase.user_id } }
+  test 'should create purchase' do
+    assert_difference('Purchase.count') do
+      post purchases_url,
+           params: { purchase: { amount: @purchase.amount, category_id: @purchase.category_id, name: @purchase.name,
+                                 user_id: @purchase.user_id } }
     end
 
```

Comment:
- [ ] It would be better if we are able to run the integration test. I could see controller tests written in the `test` directory instead of the `spec`. I have tried `rspec spec` it did not run the integration test so I had to be a little creative so i did `rspec test` that also did not work for the `integration test`